src/main/java/core/basesyntax/strategy/AddOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+
+public class AddOperationHandler implements OperationHandler {
+
+    @Override
+    public int apply(Transaction transaction) {
```

Comment:
You don't do additional action with returned quantity, fix in each handler
```suggestion
    public void handle(Transaction transaction) {
```